packages/nodes-base/nodes/ActiveCampaign/ConnectionDescription.ts

```diff
@@ -74,7 +74,7 @@ export const connectionFields: INodeProperties[] = [
 		description: 'The name of the service',
 	},
 	{
-		displayName: 'External accout ID',
+		displayName: 'External Accout ID',
```

Comment:
Out of scope but quick to fix.
```suggestion
				displayName: 'External Account ID',
```